### PR TITLE
[1LP][RFR] Nav work around for 5.11.0.16

### PR DESCRIPTION
**Pull request by digitronik**
Signed-off-by: Nikhil Dhandre <ndhandre@redhat.com>

## Purpose or Intent

In latest build `MyServiceView` is broken due to `VeriticalNav`; It will break all tests related to `service.` `currently_selected` returning value like  `['Services', 'My Services', 'Services', 'Providers']`. 

some local checks
```
ipdb> self.navigation.currently_selected                                          
['Services', 'My Services', 'Services', 'Providers']
ipdb> self.navigation.currently_selected                                          
['Storage', 'Block Storage', 'Managers']
ipdb> self.navigation.currently_selected                                          
['Services', 'Catalogs']
```
vertical_nav_bar

### PRT Run

{{pytest: cfme/tests/automate/custom_button/test_service_vm_custom_button.py::test_custom_button_with_dynamic_dialog_vm -v}}

### CODE REVIEW

File: cfme/services/myservice/ui.py [hunk 1]
```diff
@@ -55,9 +55,12 @@ class MyServicesView(BaseLoggedInPage):
 
     @property
     def in_myservices(self):
+        # Adding workaround for 5.11.0.16.
+        # Its collecting ['Services', 'My Services', 'Services', 'Providers']
         return (
             self.logged_in_as_current_user and
-            self.navigation.currently_selected == ["Services", "My Services"])
+            self.navigation.currently_selected[0:2] == ["Services", "My Services"]
```
Comment (jaryn) [inline cfme/services/myservice/ui.py]: Well this is quite minor, but you only need `self.navigation.currently_selected[:2]` there

Comment (digitronik) [reply cfme/services/myservice/ui.py]: yup, but its ok in case of indexing

File: cfme/services/myservice/ui.py [hunk 2]
```diff
@@ -55,9 +55,11 @@ class MyServicesView(BaseLoggedInPage):
 
     @property
     def in_myservices(self):
+        # Slicing in currently_selected is workaround for BZ-1733489
```
Comment (jawatts) [inline cfme/services/myservice/ui.py]: I think i may be good to do BZ(1733489).blocks here? Thoughts?

Comment (digitronik) [reply cfme/services/myservice/ui.py]: why we need blocks?
we are just limiting to index 1. If BZ fix anyway it will work fine.

**Comment by digitronik** [comment]
Hey Justin... Added please have glance... we need it fast :+1:

**Pull request merged**